src/Parser/Druid/Feral/Modules/Bleeds/RakeSnapshot.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import Snapshot, { JAGGED_WOUNDS_MODIFIER, PANDEMIC_FRACTION } from '../FeralCore/Snapshot';
+
+/*
+Identify inefficient refreshes of the Rake DoT:
+  Refresh of a Rake doing double damage due to Prowl with one that will not do double damage.
+  Early refresh of a Rake doing bonus damage from snapshot buffs with one that will do less damage.
+*/
+
+const RAKE_BASE_DURATION = 15000;
+class RakeSnapshot extends Snapshot {
+  rakeCastCount = 0;        // count of rake casts of all kinds
+  prowlLostCastCount = 0;   // count of rake buffed with Prowl ending early due to refresh without Prowl buff
+  prowlLostTimeSum = 0;     // total time cut out from the end of prowl-buffed rake bleeds by refreshing early (milliseconds)
+  downgradeCastCount = 0;   // count of rake DoTs refreshed with weaker snapshot before pandemic
+
+  hasBloodtalonsTalent = false;
+
+  on_initialized() {
+    this.spellCastId = SPELLS.RAKE.id;
+    this.durationOfFresh = RAKE_BASE_DURATION;
+    this.isProwlAffected = true;
+    this.isTigersFuryAffected = true;
+
+    const combatant = this.combatants.selected;
+    if (combatant.hasTalent(SPELLS.BLOODTALONS_TALENT.id)) {
```

Comment:
No need to do this check as `this.hasBloodtalonsTalent` isn't used and `this.isBloodtalonsAffected` can always be true (if you don't have the talent you just wont ever get the buff).